lib/active_merchant/billing/gateways/stripe_payment_intents.rb

```diff
@@ -102,6 +105,12 @@ def create_setup_intent(payment_method, options = {})
         commit(:post, 'setup_intents', post, options)
       end
 
+      def retrieve_setup_intent(seti)
```

Comment:
This looks like net-new functionality. Could you add more detail on how this would be used in an off-session workflow? The only test below looks like it checks to see that the required_action is present and that network_id is `nil`.

Reply:
Weak preference here for uncontracted names, `seti` > `setup_intent`

Reply:
I added a comment with the context in which this is used. Also updated the parameter name